docs/reference/modules/settings.md

```diff
@@ -59,11 +60,102 @@ module.exports = {
 
 ```
 
-<template v-slot:cation>
+<template v-slot:caption>
   /modules/@apostrophecms/settings/index.js
 </template>
 </AposCodeBlock>
 
+### Requirements
+
+All fields must be part of the `@apostrophecms/user` schema, except `adminLocale` that will be added automatically to
+`@apostrophecms/user` if the `adminLocales` option is set in the `@apostrophecms/i18n` module.
+
+<AposCodeBlock>
+
+``` javascript
+module.exports = {
+  fields: {
+    add: {
+      firstName: {
+        type: 'string',
+        label: 'First Name'
+      },
+      lastName: {
+        type: 'string',
+        label: 'Last Name'
+      },
+      displayName: {
+        type: 'string',
+        label: 'Display Name'
+      }
+    },
+    group: {
+      account: {
+        label: 'Account',
+        fields: [
+          'firstName',
+          'lastName',
+          'displayName'
+        ]
+      }
+    }
+  }
+};
+
+```
+
+<template v-slot:caption>
+  /modules/@apostrophecms/user/index.js
+</template>
+</AposCodeBlock>
+
+<AposCodeBlock>
```

Comment:
We could consider making this shorter, in case the user wants to copy-paste there is less to toss if it doesn't pertain to their project.